examples/demo-rollup/tests/all_tests.rs

```diff
@@ -1,6 +1,3 @@
-// For now, exclude the evm from the bank tests
-#[cfg(not(feature = "experimental"))]
 mod bank;
-#[cfg(feature = "experimental")]
```

Comment:
this flags should stay as they were, but CI should get `experimental` feature flag when invoking `cargo llvm-cov --locked --lcov --output-path lcov.info`

Reply:
We used import `sov-demo-rollup = { path = ".", features = ["experimental"] }` in tests, but it was reamoved (I think by accident). I am adding it back.

Reply:
I believe it is more practical to have the tests for the `evm` enabled as the default. This  allows other developers to quickly detect any issues they may have caused without having to rely on the CI. Additionally, while it's acceptable to have incomplete documentation for experimental features, if someone has taken the time to write a test, we should make sure it functions correctly under all conditions.